reactor-core/src/test/java/reactor/core/publisher/FluxFlatMapIterableOnDiscardShouldNotLeakTest.java

```diff
@@ -0,0 +1,20 @@
+package reactor.core.publisher;
+
+import org.reactivestreams.Publisher;
+import reactor.test.publisher.TestPublisher;
+
+import java.util.Arrays;
+
+public class FluxFlatMapIterableOnDiscardShouldNotLeakTest extends AbstractOnDiscardShouldNotLeakTest {
+
+    public FluxFlatMapIterableOnDiscardShouldNotLeakTest(boolean conditional, boolean fused) {
+        super(conditional, fused);
+    }
+
+    @Override
+    protected Publisher<Tracked<?>> transform(TestPublisher<Tracked<?>> main, TestPublisher<Tracked<?>>... additional) {
+        return main
```

Comment:
nit: looking at other similar tests, we could introduce `AbstractFluxOnDiscardShouldNotLeakTest` that defines `transform(Flux, Flux)`

Reply:
Makes sense to do

Reply:
fixed